src/components/Button/Loader/index.js

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import { Dot, LoaderWrapper, ContentWrapper, DotsWrapper } from "./styles";
+import { BUTTON_VARIANTS } from "../../constants";
+
+const Loader = ({ children, loading, variant }) => (
+  <LoaderWrapper>
+    {loading && (
+      <DotsWrapper data-bdd="loader">
+        <Dot variant={variant} />
+        <Dot variant={variant} />
+        <Dot variant={variant} />
+      </DotsWrapper>
+    )}
+    <ContentWrapper isLoading={loading}>{children}</ContentWrapper>
+  </LoaderWrapper>
+);
+
+Loader.propTypes = {
+  loading: PropTypes.bool,
```

Comment:
we could rename this props to isLoading as well. I don't think this is a blocker to merge the PR though.

Reply:
Updated.